charts/opensds/templates/deployment.yaml

```diff
@@ -52,9 +52,6 @@ spec:
               mountPath: /etc/opensds
             - name: ceph-conf-dir
               mountPath: /etc/ceph
-            - name: tgt-conf-dir
```

Comment:
Can you explain why to remove this mounted dir? I added it when I tested with lvm backend.

Reply:
While testing for Ceph backend, we have removed that portion as it was giving error while installing opensds helm chart. Now I have made few changes in deployment.yml so that it won't create any issue while using any backend (i.e. Ceph and lvm)